src/components/Layout/Layout.scss

```diff
@@ -661,10 +661,26 @@ header {
         font-size: 1.25em;
         line-height: 1.75em;
         opacity: 0.9;
+        li {
+            font-size: 1em;
+        }
```

Comment:
This fixes font sizes in nested lists

![Screen Shot 2021-09-27 at 18 46 27](https://user-images.githubusercontent.com/328367/134959367-7b1f6c57-fc31-4e32-aca0-5a863ca363a7.png)

![Screen Shot 2021-09-27 at 18 46 36](https://user-images.githubusercontent.com/328367/134959376-ba21889b-dcb3-4774-bf91-167ddbaccf94.png)

Reply:
@smallbrownbike - you okay with this?